scss/_button.scss

```
@@ -0,0 +1,39 @@
+.button {
+  background-color: var(--accent);
+  border: none;
+  position: relative;
+  padding: 20px 32px;
+  min-width: 210px;
+  min-height: 60px;
+  &__text {
+    position: absolute;
+    z-index: 2;
+    left: 0;
+    right: 0;
+    top: 20px;
+    color: var(--white);
+  }
```

чтоб работал **z-index**, можно поставить `position: relative`
```suggestion
  &__text {
    position: relative;
    z-index: 2;
    color: var(--white);
  }
```